### ISSUE TITLE
Additive random mutation with `gene_space` `low` and `high`

### ISSUE THREAD

**Issue by theo-brown** [ahmedfgad/GeneticAlgorithmPython#229]
In the [docs](https://pygad.readthedocs.io/en/latest/pygad.html#how-mutation-works-with-the-gene-space-parameter), it says that:

> If a gene has its static space defined in the gene_space parameter, then mutation works by replacing the gene value by a value randomly selected from the gene space.

However, if a `gene_space` is defined as a dict with `low` and `high` parameters, it should be possible to generate additive mutations, so that `new_value = old_value + random_value`, rather than `new_value = random_value`.

I had a brief look in the source code, and this wasn't supported at the moment.

**Comment by theo-brown** [comment]
My default approach to this would be to use rejection sampling and Gaussian noise, but I'm not sure how well it aligns with the methods currently used for additive random mutations in PyGAD, which seem to rely on uniform sampling from a fixed range.

1. Add new parameter, `mutation_noise_variance`
2. Generate `noise ~ N(0, 1)`
3. Set `candidate_value = old_value + mutation_noise_variance*noise`
4. Set `new_value = candidate_value` if `low < candidate_value < high` else repeat from 2

**Comment by ahmedfgad** [comment]
You are right. But this should be supported without adding additional parameters. The 2 parameters:
1. `random_mutation_min_val`
2. `random_mutation_max_val`

should be used to generate the random value.
